source/Planet.cpp

```diff
@@ -169,6 +178,8 @@ void Planet::Load(const DataNode &node)
 			text += value;
 			text += '\n';
 		}
+		else if(key == "landing speed")
+			landingSpeed = LANDING_SPEED[value];
 		else if(key == "government")
```

Comment:
You should probably add a warning if a string is given that isn't a valid speed.